docker/solver/README.rst

```diff
@@ -14,20 +14,20 @@ Prerequisites
    git clone https://github.com/pyansys/pydyna.git
    cd pydyna
 
-  The ``docker`` file in the the ``docker/solver`` directory is used to build the
+  The ``docker`` file in the  ``docker/pre`` directory is used to build the
```

Comment:
```suggestion
  The ``docker`` file in the  ``docker/solver`` directory is used to build the
```

Reply:
You are editing the README in the ``solver`` folder so I am guessing this should be solver...